src/app/search/search.component.html

```diff
@@ -12,7 +12,7 @@ <h2 class="heading-h2" aria-label="Criteria">Criteria</h2>  <!--  search-filter
   </div>
   <div class="search-right" id="search-result" aria-live="polite" role="region" >
     <ccd-search-result *ngIf="resultView" [jurisdiction]="jurisdiction" [caseType]="caseType" [caseState]="caseState" [resultView]="resultView"
-                       [caseFilterFG]="caseFilterFG" [paginationMetadata]="paginationMetadata"
```

Comment:
Are we sure we want to remove caseFilterFG? I think it is needed.

Reply:
done